### DISCUSSION TITLE
Amasty Megamenu

### DISCUSSION THREAD

**Discussion by lcastro-art** [breezefront/community#70] [3. Integrations]
Hello,

I have installed the Breeze template on Magento 2.4.4, and I have some Amasty plugins such as Megamenu, etc. How can I integrate these plugins to work with the "Enable Breeze Experience" function? Would it be possible for you to show us how to do it?

**Comment by dimdimich** [comment]
Thank you for choosing Breeze as your Magento 2 frontend solution. We recommend contacting Amasty, the provider of the modules you are using, to request that they add Breeze compatibility. You can use the sample contact message provided on our blog at https://swissuplabs.com/blog/breeze-frontend-for-magento-2-needs-your-help-join-in-improving-breeze-compatibility/.

The more people that request Breeze compatibility from Amasty, the higher the chances that they will prioritize adding it to their modules. We have received multiple inquiries about this matter this week alone and hope that Amasty will take it into consideration.

In the event that Amasty refuses to add Breeze compatibility, please refer to our integration tips on https://breezefront.com/docs/integration-tips. If you have any questions related to integrating Breeze, our support forum is available to assist you.

**Comment by vovayatsyuk** [comment]
### :loudspeaker:  For everyone interested in this kind of module.

We have our own menu builder module and it's working with breeze theme: https://swissuplabs.com/magento-2-navigation-pro-extension.html

**Comment by yasser-moudouani** [comment]
Hello,
 
Did you have any feedback regarding the Breeze compatibility of the Amasty Mega Menu module?
We start working on Breeze theme integration and we want to anticipate any potential conflicts before proceeding.
